src/test/java/org/folio/des/scheduling/ExportTriggerTest.java

```diff
@@ -62,6 +68,9 @@ class ExportTriggerTest {
   @MockBean private ExportConfigValidatorResolver exportConfigValidatorResolver;
   @MockBean private JobCommandBuilderResolver jobCommandBuilderResolver;
   @MockBean private KafkaService kafka;
+  @MockBean private ConfigurationClient client;
+  @MockBean JobServiceImpl jobService;
```

Comment:
if all other fields are private - it makes sence to do the same with JobServiceImpl

Reply:
fixed.